bin/upload-istioctl

```
@@ -43,7 +43,7 @@ if [[ ${RELEASE} == true ]]; then
   LDFLAGS=""
   while read line; do
     read SYMBOL VALUE < <(echo $line)
-    LDFLAGS=${LDFLAGS}" -X istio.io/pilot/cmd/version.${SYMBOL}=${VALUE}"
+    LDFLAGS=${LDFLAGS}" -X istio.io/pilot/tools/version.${SYMBOL}=${VALUE}"
```

I think it was silently failing. No error printed no warning.